web-admin/README.md

```
@@ -141,11 +152,13 @@ Pour maintenir une organisation claire du code, les fonctions spécifiques aux p
 
 ## Sécurité
 
 - Toutes les entrées utilisateur doivent être validées et nettoyées
 - L'authentification est gérée via des sessions sécurisées
-- Les mots de passe sont hachés avec bcrypt
+- Les mots de passe sont hachés avec SHA-256
 - Protection contre les attaques XSS et CSRF implémentée
-- Les requêtes à l'API nécessitent une authentification par jeton 
\ No newline at end of file
+- Les requêtes à l'API nécessitent une authentification par jeton
+- Journalisation des événements de sécurité 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)